mitodl/hugo-course-publisher
# Basic theme

#### What are the relevant tickets?
Fixes https://github.com/mitodl/hugo-course-publisher/issues/2

#### What's this PR do?
Adds a basic, blank theme to the project called "hugo-course."

#### How should this be manually tested?
Pull down the branch and run the site locally using `docker-compose up --build` and then visit http://localhost:3000.  You should see the test "Hello world!" content coming from the site's layout, and the header and footer that come from the layout which currently contain links to the root page.
